jerry-math/expm1.c

```diff
@@ -265,7 +265,10 @@ expm1 (double x)
       y = one - (e - x);
       if (k == 1024)
       {
-        y = y * 2.0 * 0x1p1023;
+        double_accessor const_0x1p1023;
+        const_0x1p1023.as_int.hi = 0x7fe00000;
+        const_0x1p1023.as_int.lo = 0;
+        y = y * 2.0 * const_0x1p1023.dbl;
```

Comment:
I'd prefer a slightly different approach:
```c
        const double twop1023 = ((double_accessor) { .as_int = { .hi = 0x7fe00000, .lo = 0 } }).dbl; // 0x1p1023
        y = y * 2.0 * twop1023;
```

Reply:
not works on old msvc，

Reply:
That will get problematic at some point. Please, specify what old means. JerryScript is a C99 project. We will have to draw the line somewhere, what deviations from the standard do we still tolerate. MSVC starts to build up quite a list.

Reply:
> That will get problematic at some point. Please, specify what old means. JerryScript is a C99 project. We will have to draw the line somewhere, what deviations from the standard do we still tolerate. MSVC starts to build up quite a list.

you are right, works:)

Reply:
> I'd prefer a slightly different approach:
> 
> ```c
>         const double twop1023 = ((double_accessor) { .as_int = { .hi = 0x7fe00000, .lo = 0 } }).dbl; // 0x1p1023
>         y = y * 2.0 * twop1023;
> ```

./jerry-math/expm1.c:268: error: brace should be placed on a separate line
Error: Process completed with exit code 1.

single line are invalid style